python/pyiceberg/table/base.py

```diff
@@ -19,8 +19,8 @@
 
 from pydantic import Field
 
-from pyiceberg.catalog.base import Identifier
 from pyiceberg.table.metadata import TableMetadataV1, TableMetadataV2
+from pyiceberg.typedef import Identifier
```

Comment:
@samredai, should we also move this to either `table.py` or `table/__init__.py`?

Reply:
Done! https://github.com/apache/iceberg/pull/5458